src/client/Microsoft.Identity.Client/Platforms/Android/AndroidHttpClientFactory.cs

```diff
@@ -17,7 +17,7 @@ public HttpClient GetHttpClient()
 
             var httpClient = new HttpClient(
                 // As per Xamarin guidance https://docs.microsoft.com/en-us/xamarin/android/app-fundamentals/http-stack?tabs=windows
-                new Xamarin.Android.Net.AndroidClientHandler());
+                new Xamarin.Android.Net.AndroidMessageHandler());
```

Comment:
Won't this break older Xamarin applications that target sdk<12
https://docs.microsoft.com/en-us/dotnet/api/xamarin.android.net.androidmessagehandler?view=xamarin-android-sdk-12

Reply:
@SameerK-MSFT - shouldn't this be added only to MAUI?